Makefile

```diff
@@ -0,0 +1,158 @@
+VIRTUALENV=~/.venvs/pixua
+
+.PHONY: setup
+setup: requirements install
+
+.PHONY: requirements
+requirements: requirements_py requirements_js
+	@echo "Installed requirements"
+
+.PHONY: install
+install: requirements install_py install_js
```

Comment:
In that case, I'll keep the install because it is more intuitive for people used to make. so we'll keep ./go build (for compatibility reasons) but have make install for the future